generated/nidcpower/nidcpower.proto

```diff
@@ -1281,12 +1386,12 @@ message ErrorMessageResponse {
 
 message ErrorQueryRequest {
```

Comment:
I think this was wrong before. The ErrorQuery method should return the error_code and error_message.